### ISSUE TITLE
Fugu 14 app did Jailbreak + untether, but unc0ver says "Unsupported".

### ISSUE THREAD

**Issue by Tim2223** [LinusHenze/Fugu14#176]
![IMG_0880](https://user-images.githubusercontent.com/93207233/139940457-2ec71ad6-2553-4264-9110-c59b0f1822a0.PNG)
![IMG_0879](https://user-images.githubusercontent.com/93207233/139940537-b4230f62-5a76-425c-b18d-d640e6ccd366.PNG)

**Comment by jaygee562** [comment]
you did altsotre method? or you compile fugu yourself w xcode?

**Comment by Tim2223** [comment]
Yes, I re-compiled it by myself, and I did try the altstore and sideloadly methods, but have many times its say "Unsupported" yet, and Fugu 14 app couldn't get me to rootfs

**Comment by Shulkk** [comment]
what iOS version are you on?

**Comment by Tim2223** [comment]
14.4.1

**Comment by ScknCbt** [comment]
You put your phone into safemode via irecvery?

**Comment by Tim2223** [comment]
No, I never do that

**Comment by Tim2223** [comment]
I was turned into recovery mode then following irecovery commands and reboot, but Fugu 14 does stopped work

**Comment by ScknCbt** [comment]
You have a old version of unc0ver. Just delete Altstore and unc0ver and sideload the new version and try it again

**Comment by Tim2223** [comment]
I installed the new version of unc0ver jailbreak before run again with Fugu 14 setup and jailbreak + untether, didn't work and still it showing "Unsupported"

**Comment by RiskyAppleJuice** [comment]
Just reboot your iPhone. And come back to uncover again. That's what I did. And it works. Both Xcode or AltStore way

**Comment by Tim2223** [comment]
It doesn't work after reboot

**Comment by ni94wei1** [comment]
![QQ图片20211105100313](https://user-images.githubusercontent.com/25652988/140477887-4c1d86ac-d8de-4d80-962a-f5a6b75d5264.jpg)
i need help

**Comment by ni94wei1** [comment]
delete Altstore and unc0ver and sideload the new version  
Jailbreak complete

**Comment by putra1990** [comment]
done
i same error ipad mini 5 ios 14.4.2 unsupported

**Comment by putra1990** [comment]
succes with altstore
anti virus disable
